map/framework.cpp

```diff
@@ -341,6 +341,7 @@ void Framework::Migrate(bool keepDownloaded)
   InitUGC();
   InitSearchAPI();
   InitCityFinder();
+  InitDiscoveryManager();
```

Comment:
It looks like DiscoveryManager must be destoyed after search api, not before.

Reply:
it must be destroyed before search api, there is a mistake in the code.